.github/workflows/tests.yml

```diff
@@ -0,0 +1,156 @@
+name: Tests
+
+on: [push]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: 'npm'
+      - run: npm ci
+      - run: npm run bootstrap -- --ci
+      - run: npm run lint
+
+  puppeteer:
+    strategy:
+      matrix:
+        node: [16, 18, 20]
+    runs-on: ubuntu-latest
+    needs: lint
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node}}
+          cache: 'npm'
+      - run: npm run build --prefix=packages/puppeteer
```

Comment:
Most jobs below this line are missing an `npm ci` step after `actions/setup-node`